### PR TITLE
Final Evaluation Blog Post for Google Summer of Code

### PR DESCRIPTION
**Pull request by SidharthBansal**
No related image with this post

### CODE REVIEW

**Comment by davidevaleriani** [comment]
Waiting for an approval by @tacaswell or @story645 before merging

File: content/posts/GSoC_Coding_Phase_Blog_5/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Adopting reusability and Do not Repeat Yourself (DRY) Principles
+
+Later, we refactored the `matplot_baseline_image_generation` and `generate_missing` command line flags to single command line flag `matplotlib_baseline_image_generation` as the logic was similar for both of them. Now, the image generation on the time of fresh install of matplotlib and the generation of missing baseline images works with the `python3 -pytest lib/matplotlib matplotlib_baseline_image_generation` for the `lib/matplotlib` folder and `python3 -pytest lib/mpl_toolkits matplotlib_baseline_image_generation` for the `lib/mpl_toolkits` folder.
+
+## Writing the documentation
+
+We have written documentation explaining the following scenarios:
+1. How to generate the baseline images on a fresh install of matplotlib?
+2. How to generate the missing baseline images on fetching changes from master?
```
Comment (tacaswell) [inline content/posts/GSoC_Coding_Phase_Blog_5/index.md:27]: Add a bullet point for "how to intentionally change an image"?

**Review by tacaswell — APPROVED** [review]

**Comment by tacaswell** [comment]
Left one comment, but happy for this to be published as-is.

File: content/posts/GSoC_Coding_Phase_Blog_5/index.md (hunk 1 above)
Comment (SidharthBansal) [reply content/posts/GSoC_Coding_Phase_Blog_5/index.md:27]: Done! Thanks

**Comment by SidharthBansal** [comment]
PR is ready to merge!!!

**Pull request merged**